package-lock.json

```diff
@@ -21026,9 +20906,9 @@
       }
     },
     "terser": {
-      "version": "5.14.2",
-      "resolved": "https://registry.npmjs.org/terser/-/terser-5.14.2.tgz",
-      "integrity": "sha512-oL0rGeM/WFQCUd0y2QrWxYnq7tfSuKBiqTjRPWrRgB46WD/kiwHwF8T23z78H6Q6kGCuuHcPB+KULHRdxvVGQA==",
+      "version": "5.14.1",
```

Comment:
Do not downgrade `terser`

Reply:
this change is coming automatically after running 'npm install'

Reply:
5.14.1 triggers security alerts like [Terser insecure use of regular expressions before v4.8.1 and v5.14.2 leads to ReDoS #15](https://github.com/microsoft/powerplatform-vscode/security/dependabot/15), hence why it was bumped to 5.14.2 in #242.

`npm install` modifies the lock file, so you should be using `npm ci` ("Clean Install") when you are not intending to modify the lockfile.